src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -7954,11 +7954,34 @@ public void CS0501ERR_ConcreteMissingBody02()
     protected abstract object S { set; } // no error
 }
 ";
-            CreateCompilation(text).VerifyDiagnostics(
+            // IMPORTANT: There used to be CS0501 (missing body) error on P and Q properties.
+            // This has changed with semi auto properties, and it was decided (for all lang versions) to keep
+            // the new logic for determining if a property is auto property or not.
+            // See https://github.com/dotnet/roslyn/pull/57076#discussion_r753751334
+
+            // PROTOTYPE(semi-auto-props): Unassigned field warning?
+
+            CreateCompilation(text, parseOptions: TestOptions.Regular10).VerifyDiagnostics(
```

Comment:
>VerifyDiagnostics [](http://example.com/codeflow?start=73&length=17)

Unless I am missing something, given that supporting mixed accessors is not a goal for the initial PR, I do not expect any changes for this test. However, if baseline changes are necessary, please include comments with corresponding source lines.